src/material-experimental/mdc-helpers/_focus-indicators.scss

```diff
@@ -64,25 +64,16 @@
     border-radius: 50%;
   }
 
-  // Render the focus indicator on focus. Defining a pseudo element's
-  // content will cause it to render.
-
-  // For checkboxes, radios and slide toggles, render the focus indicator when we know
-  // the hidden input is focused (slightly different for each control).
-  .mdc-checkbox__native-control:focus ~ .mat-mdc-focus-indicator::before,
-  .mat-mdc-slide-toggle-focused .mat-mdc-focus-indicator::before,
-  .mat-mdc-radio-button.cdk-focused .mat-mdc-focus-indicator::before,
-
-  // For buttons and list items, render the focus indicator when the parent
-  // button or list item is focused.
-  .mat-mdc-button-base:focus .mat-mdc-focus-indicator::before,
-  .mat-mdc-list-item:focus > .mat-mdc-focus-indicator::before,
-
-  // For options, render the focus indicator when the class .mat-mdc-option-active is present.
-  .mat-mdc-focus-indicator.mat-mdc-option-active::before,
-
-    // For all other components, render the focus indicator on focus.
-  .mat-mdc-focus-indicator:focus::before {
+  // Render a focus indicator when either:
```

Comment:
If we want, we can simplify this to just the first two conditions, we'd just need options to toggle the class `.mat-mdc-focus-indicator-render` on active/non-active (and same for non-MDC implementations). I'm fine either way. In my head, `.mat-mdc-option-active` / `.mat-active` is like the `aria-activedescendant` class.

Reply:
As a related note: It might also make sense to rename `.mat-mdc-option-active` to `.mat-mdc-active` (so that it can be eventually used as an "`aria-activedescendant` class" on non-option elements).